command/cp.go

```diff
@@ -429,7 +429,13 @@ func (c Copy) Run(ctx context.Context) error {
 	}
 
 	for object := range objch {
-		if object.Type.IsDir() || errorpkg.IsCancelation(object.Err) {
+		if errorpkg.IsCancelation(object.Err) || object.Type.IsDir() {
+			continue
+		}
+		if !object.Type.IsRegular() {
+			err := fmt.Errorf("object '%v' is skipped due to it's a special file", object)
```

Comment:
I think we can omit the `skip` in the message, object errors here does not cause execution to stop anyways.
```suggestion
			err := fmt.Errorf("object '%v' is is a special file", object)
```
or;
```suggestion
			err := fmt.Errorf("object '%v' is not a regular file", object)
```

My vote is on the latter, what do you think?

Reply:
I preferred the second one. https://github.com/peak/s5cmd/pull/618/commits/395b116fa6be857396bacf116c6caa71b35b10c2